trio/_core/_run.py

```diff
@@ -1240,11 +1256,11 @@ def print_stack_for_task(task):
             if hasattr(coro, "cr_frame"):
                 # A real coroutine
                 yield coro.cr_frame, coro.cr_frame.f_lineno
-                coro = coro.cr_await
+                coro = coro.cr_await  # type: ignore # TODO
             elif hasattr(coro, "gi_frame"):
                 # A generator decorated with @types.coroutine
                 yield coro.gi_frame, coro.gi_frame.f_lineno
-                coro = coro.gi_yieldfrom
+                coro = coro.gi_yieldfrom  # type: ignore # TODO
```

Comment:
idk if these are an issue, but since the signature is now typed the type checker complains about them.

Reply:
Maybe just type `coro: Any` to effectively disable type-checking, since this is doing a bunch of dynamic introspection, and would accept objects that duck-type as coroutines/generators (Cython)...

Reply:
Sure, could be made stricter later on if somebody wants.